src/components/veNft/veNftCarousel.tsx

```diff
@@ -1,5 +1,114 @@
-const veNftCarousel = () => {
-  return <div>veNftCarousel</div>
+import { Row, Col, Image, FormInstance } from 'antd'
+import { VeNftVariant } from 'models/v2/veNft'
+import { getVeNftBaseImage } from 'utils/v2/veNft'
+
+type StakingNFTCarouselProps = {
+  tokensStaked: string
+  variants: VeNftVariant[]
+  baseImagesHash: string
+  form: FormInstance
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  tokenMetadata: any
```

Comment:
**question** is this really `any`? Is it possible to define it?

Reply:
Made type for this and forgot to retype here. Done.